src/main/java/picard/illumina/IlluminaBasecallsToFastq.java

```diff
@@ -60,7 +55,7 @@
         programGroup = BaseCallingProgramGroup.class
 )
 @DocumentedFeature
-public class IlluminaBasecallsToFastq extends CommandLineProgram {
+public class IlluminaBasecallsToFastq extends ExtractBarcodesProgram {
     static final String USAGE_SUMMARY = "Generate FASTQ file(s) from Illumina basecall read data.  ";
     static final String USAGE_DETAILS = "<p>This tool generates FASTQ files from data in an Illumina BaseCalls output directory.  " +
```

Comment:
Should probably update the details to discuss on-the-fly demux and which parameters affect it.